### ISSUE TITLE
doesnt open multiple robloxs

### ISSUE THREAD

**Issue by robloxUser827** [ic3w0lf22/Roblox-Account-Manager#207]
i just got on today wanting to use it and it doesn't work how sad, please fix it immediately

**Comment by Bvcv12** [comment]
I agree with this. Even when opening RAMSettings.ini, then adding EnableMultiRbx=true under the [General] section it still doesn't work for me. Even when that part is not automatically deleted, when I try opening a new instance of Roblox, the new one just replaces the old one.

**Comment by robloxUser827** [comment]
lemme try the enable multiple robloxs thing rq

**Comment by robloxUser827** [comment]
nvm when i open the manager when roblox is open it just deletes the manager
